gapic/src/main/com/google/gapid/models/ProfileExperiments.java

```diff
@@ -16,21 +16,23 @@
 package com.google.gapid.models;
 
 import com.google.gapid.proto.service.Service;
+import com.google.gapid.proto.service.path.Path;
+
+import java.util.List;
 
 public final class ProfileExperiments {
   public final boolean disableAnisotropicFiltering;
+  public final List<Path.Command> disabledCommands;
 
-  public ProfileExperiments() {
-    this(false);
-  }
```

Comment:
Keep the default constructor to create an "empty" one, so you don't have to keep changing all the places where you create a default instance.